README.md

```diff
@@ -56,6 +61,14 @@ As a requirement, please make sure to:
 
 Run `make dev` to get a working environment in k8s
 
+### OpenFGA initialization
+
+The Admin Service comes up with authorization disabled (see `AUTHORIZATION_ENABLED` env var), The env vars `OPENFGA_AUTHORIZATION_MODEL_ID` and `OPENFGA_STORE_ID` which are needed for the correct functioning of the RBAC APIs get set by the job `admin-ui-openfga-setup`, after this has completed a developer is supposed to bounce the deployment to get the application to source the new env vars, setting `AUTHORIZATION_ENABLED` will make sure those endpoints use OpenFGA as a backend instead of a `NoOp implementation` (behaviour will change,  see https://github.com/canonical/identity-platform-admin-ui/issues/221)
+
+As k8s jobs on their own don't get deleted, make sure that to run `kubectl delete job admin-ui-openfga-setup` if you are planning to change the model, make sure environment variables in the `identity-platform-admin-ui` configmap reflect the status you want
```

Comment:
Again to make it clearer we should provide the steps that are needed in order to change the model. Something like

```
If you wish to make changes to the openfga model, you need to make sure that the job for setting up openfga is deleted before redeploying the the admin UI:

kubectl delete job admin-ui-openfga-setup
make dev
```